### ISSUE TITLE
Type Error on `git feature`

### ISSUE THREAD

**Issue by eykd** [trydionel/git-pivotal#2]
```
$ git feature
/Library/Ruby/Gems/1.8/gems/git-pivotal-0.2.0/bin/../lib/pivotal/attributes.rb:14:in `has_attributes': wrong argument type Symbol (expected Proc) (TypeError)
        from /Library/Ruby/Gems/1.8/gems/git-pivotal-0.2.0/bin/../lib/pivotal/story.rb:4
        from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
        from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `require'
        from /Library/Ruby/Gems/1.8/gems/git-pivotal-0.2.0/bin/../lib/pivotal.rb:6
        from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
        from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `require'
        from /Library/Ruby/Gems/1.8/gems/git-pivotal-0.2.0/bin/git-feature:5
        from /usr/bin/git-feature:19:in `load'
        from /usr/bin/git-feature:19
```

This is with a fresh installation of git-pivotal, with pivotal.project-id, pivotal.api-token, and pivotal.full-name all set correctly, on my first (and subsequent) attempt at using `git feature`.

Using standard system Ruby on OS X Leopard: 
    ruby 1.8.6 (2008-08-11 patchlevel 287) [universal-darwin9.0]

**Comment by trydionel** [comment]
Thanks for the note eykd.  I was relying on a feature introduced in Ruby 1.8.7 (Symbol to Proc shorthand) and had inadvertently prevented the library from working in 1.8.6.  I've solved the particular problem you reported and released a new version (v0.2.1).  Please let me know if you see any other problems.  Thanks!

**Comment by eykd** [comment]
That appears to be working. Thanks for the quick fix!
